### PR TITLE
made workspaces model

### PR DESCRIPTION
**Pull request by Tom-Apted**
added workspaces model, table in database and controller with index and show methods with empty views @AJM1994 @MPRevell @gougendou

### CODE REVIEW

File: db/migrate/20230227123942_create_workspaces.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+class CreateWorkspaces < ActiveRecord::Migration[7.0]
+  def change
+    create_table :workspaces do |t|
+      t.string :name
+      t.float :price
+      t.string :address
+      t.string :photo
```
Comment (gougendou) [inline db/migrate/20230227123942_create_workspaces.rb:7]: Ultimately we will delete this field and use ActiveStorage, but for now this is fine

File: app/controllers/workspaces_controller.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+class WorkspacesController < ApplicationController
+  def index
+    @workspace = Workspace.find(params[:id])
```
Comment (MPRevell) [inline app/controllers/workspaces_controller.rb:3]: I think we need to change these around, plural with the index and vice versa (as we are listing all instances in the index view) and only one with the show route.

**Review by AJM1994 — CHANGES_REQUESTED** [review]
quick change to make :)

**Review by MPRevell — CHANGES_REQUESTED** [review]
See comment on Workspaces Controller for more info, only small changes required.

**Review by AJM1994 — APPROVED** [review]

**Review by MPRevell — APPROVED** [review]

**Comment by gougendou** [comment]
Looks great

**Pull request merged**